ttyplot.c

```diff
@@ -301,8 +301,14 @@ int main(int argc, char *argv[]) {
         exit(0);
     }
 
-    // Reset the getopt index as per spec.
-    optind = 0;
+    // Run a 2nd iteration over the arguments to actually process the options.
+    // According to getopt's documentation this is done by setting optind to 1
+    // (or 0 in some special cases). On BSDs and Macs optreset must be set to 1
+    // in addition.
+    optind = 1;
+#if defined(__OpenBSD__) || defined(__FreeBSD__) || defined(__NetBSD__) || defined(__APPLE__)
```

Comment:
I think we're about to discriminate DragonFly BSD here *unless*…
- DragenFly BSD also defines one of the other macros? (which I haven't seen indication for so far)
- We add [`__DragonFly__`](https://sourceforge.net/p/predef/wiki/OperatingSystems/#dragonfly) to the mix

What do you think?

Reply:
`__DragonFly__` check added.